src/main/scala/viper/server/vsi/HTTP.scala

```diff
@@ -68,15 +68,20 @@ trait VerificationServerHttp extends VerificationServer with CustomizableHttp {
   private var stoppedPromise: Promise[Unit] = _
   var bindingFuture: Future[Http.ServerBinding] = _
 
-  /** The server has been stopped when `stopped` completes. Afterwards, the execution context can be terminated. */
-  override def start(active_jobs: Int): Unit = {
+  /** The server has been stopped when `stopped` completes. Afterwards, the execution context can be terminated.
```

Comment:
It's slightly weird that the comments talks about stopping the server, whereas it's right above the method ```start```.

Reply:
Although that's not about this particular PR.